src/stdlib_io.fypp

```diff
@@ -81,7 +81,7 @@ module stdlib_io
 contains
 
   #:for k1, t1 in KINDS_TYPES
-    subroutine  loadtxt_${t1[0]}$${k1}$(filename, d, skiprows, max_rows)
+    subroutine  loadtxt_${t1[0]}$${k1}$(filename, d, skiprows, max_rows, fmt)
```

Comment:
[to_string](https://stdlib.fortran-lang.org/page/specs/stdlib_strings.html#to_string) actually uses `format` instead of `fmt` (I prefer `fmt`, but let see what others think about this).

Reply:
Since these functions are generally built to replicate the numpy functions of the same name, I thought it made sense to keep the naming convention used there.   savetxt uses 'fmt'.   (loadtxt doesnt' have an equivalent fmt field, only a delimiter field)